app/models/ability.rb

```diff
@@ -55,6 +55,9 @@ def create_permissions(user=nil, session=nil)
 
       if @user_groups.include? "manager"
         can :create, Admin::Collection
+        can :manage, Admin::Collection do |collection|
```

Comment:
It looks like before we didn't grant blanket manage ability to managers and instead went piecemeal but I don't remember why and can't think of a reason right now why not to do it this way.  So I guess I'm just noting this for the record.